marketplace-kit-deploy.js

```diff
@@ -7,45 +7,58 @@ const program = require('commander'),
   logger = require('./lib/logger'),
   validate = require('./lib/validators'),
   Gateway = require('./lib/proxy'),
-  deployServiceClient = require('./lib/deployServiceClient'),
+  assets = require('./lib/assets'),
   version = require('./package.json').version;
 
 const uploadArchive = (env, usingDeploymentService) => {
   const options = usingDeploymentService ? ['--without-assets'] : [];
-  const archive = spawn(command('marketplace-kit-archive'), options, {
-    stdio: 'inherit'
-  });
-
-  archive.on('close', code => {
-    if (code === 1) {
-      logger.Error('Deploy failed.');
-      process.exit(1);
-    }
-
-    const push = spawn(command('marketplace-kit-push'), [], {
-      stdio: 'inherit',
-      env: env
+  return new Promise((resolve, reject) => {
+    const archive = spawn(command('marketplace-kit-archive'), options, {
+      stdio: 'inherit'
     });
 
-    push.on('close', exitCode => {
-      if (exitCode === 1) {
+    archive.on('close', code => {
+      if (code === 1) {
         logger.Error('Deploy failed.');
-        process.exit(1);
-      } else if(exitCode === 0) {
-        process.exit(0);
+        reject(e);
       }
+
+      const push = spawn(command('marketplace-kit-push'), [], {
+        stdio: 'inherit',
+        env: env
+      });
+
+      push.on('close', (exitCode) => {
+        if (exitCode === 1) {
+          logger.Error('Deploy failed.');
+          reject(false);
+        } else if(exitCode === 0) {
+          resolve(true);
+        }
+      });
     });
   });
 };
 
+const deploy = async(env, authData, params) => {
+  await uploadArchive(env, params.directAssetsUpload);
+  if (params.directAssetsUpload){
+    const gateway = new Gateway(authData);
+    await assets.deployAssets(gateway);
+    logger.Success('Assets deployed to S3');
+  }
+};
+
+PARTNER_PORTAL_HOST = process.env.PARTNER_PORTAL_HOST || 'https://portal.apps.near-me.com';
```

Comment:
```suggestion
PARTNER_PORTAL_HOST = process.env.PARTNER_PORTAL_HOST || 'http://partners.platform-os.com';
```

Reply:
@pavelloz I've changed this address in separate PR.